server/routes/feed.go

```diff
@@ -0,0 +1,58 @@
+package routes
+
+import (
+	"localdev/main/models"
+	"localdev/main/services"
+	"net/http"
+	"encoding/json"
+	"log"
+
+	"google.golang.org/appengine"
+	"google.golang.org/api/iterator"
+	"cloud.google.com/go/datastore"
+)
+
+
+func GetPhotosForFeed(w http.ResponseWriter, r *http.Request) {
+	var response []string
+	ctx := appengine.NewContext(r)
+	query := datastore.NewQuery("Image").Order("-created").Limit(100) //sorts Images entities by creation time in descending order
+	it := services.Locator.DsClient().Run(ctx, query)
+	for {
+		var feed models.Feed
+		var image models.Image
+		_, err := it.Next(&image)
+		if err == iterator.Done {
+			break
+		}
+		if err != nil {
+			log.Printf("Error fetching next image: %v", err)
+			w.WriteHeader(http.StatusInternalServerError)
+			return
+		}
+		feed.OwnerID = image.OwnerID
+		feed.ImageURL = image.GCSObjectID()
```

Comment:
I guess it should be `"https://storage.cloud.google.com/" + config.ImageBucket() + "/" + image.GCSObjectID()`, since it's a URL